snap/local/etc/spark8t/spark-defaults.conf

```diff
@@ -1 +1 @@
-spark.kubernetes.container.image=ghcr.io/canonical/charmed-spark:3.4-22.04_stable
+spark.kubernetes.container.image=ghcr.io/canonical/charmed-spark:3.4-22.04_edge
```

Comment:
why is this being changed?

Reply:
We agreed that the monitoring jars would not be part of the stable release. Therefore, if we want to use the monitoring, this needs to be upgraded. 

I don't think it is too worrying as we release on `edge` and it is fine to point to `edge` images. It is important that before any promotion, the pointer is updated accordingly (meaning that beta releases can only point to beta, candidate or stable; and so on)